src/content/pt/2019/accessibility.md

```diff
@@ -0,0 +1,312 @@
+---
+part_number: II
+chapter_number: 9
+title: Accessibility
+description: Accessibility chapter of the 2019 Web Almanac covering ease of reading, media, ease of navigation, and compatibility with assistive technologies.
```

Comment:
You seem to have taken @HakaCode 's feedback too far 😀 He was just saying keep the "Web Almanac" brand name in English but translate the rest of this sentence.

So something like this?

```suggestion
description: Capítulo de acessibilidade do da Web Almanac de 2019 cobrindo facilidade de leitura, mídia, facilidade de navegação e compatibilidade com tecnologias assistivas.
```